pkg/remote/aws/iam_user_policy_supplier.go

```diff
@@ -3,7 +3,10 @@ package aws
 import (
 	"fmt"
 
+	error2 "github.com/cloudskiff/driftctl/pkg/remote/error"
```

Comment:
Could you do a pass to use same import name in all places ? `remoteerror` is better than `error2`